### ISSUE TITLE
Add universal folder?

### ISSUE THREAD

**Issue by danydev** [bendyworks/bwoken#59]
My UIAutomation tests, are able to run both on iPhone and iPad (I use tuneup js that adds target.isDeviceiPhone() and target.isDeviceiPad() to make them universal).

Would be really cool to have a "universal" directory where we put tests meant to be run both on iPhone and iPad.

The only option right now seems to be copying the files in both folders iphone and ipad, but I don't like it at all.

I also tried to make a folder alias inside iphone\ipad pointing to a folder where I have the tests, but it is ignored by bwoken.

If someone can point me in the right direction I could try to implement this feature, also if I'm not experienced with Ruby :-)

**Comment by dyba** [comment]
:+1: I tried creating a `lib/` folder inside the `coffeescript/` folder but I think any other folders not generated by bwoken are ignored.

**Comment by dyba** [comment]
My fault, I was `#import`ing a CoffeeScript file when I should have `#import`ed the JavaScript file:

Inside my .coffee file

``` coffeescript
#import "../../lib/some_file.js"

test "quick example", (target, app) ->
```

**Comment by danydev** [comment]
I don't think your issue is related to my feature request, I asked about adding an "universal" folder other than "ipad" and "iphone", all tests on this folder should be run for ipad and iphone.
So that I don't need to duplicate my universal tests in both folders "ipad" and "iphone"

**Comment by danydev** [comment]
@listrophy do you know some sort of workaround, or any hint to solve it?

Thanks!.

**Comment by listrophy** [comment]
hmm, I take it you've tried symbolic links already from your initial post?

Have you tried making a hard link?

**Comment by danydev** [comment]
Yeah, I tried symbolic links, and I did not try with hard link because, hard links can't be done for directories, and I don't want to make an hard link for each new test that I'll write (since the tests are meant to be supported by a team I don't want to add manual steps to the workflow).

thanks for your reply, I think that sooner or later I'll need to try to extend bwoken to support this feature. :-)

**Comment by listrophy** [comment]
Cool. Yeah, I can definitely see this as a useful feature. I've got it slated for 2.1 release, but that may get bumped further down the road because the iOS project I'm on is iPhone-only at the moment.

Of course, if someone implements it, I'd pull it into the next release regardless. :wink:
